src/test/java/org/broadinstitute/hellbender/engine/dataflow/datasources/FakeReferenceSource.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+
+import java.nio.charset.Charset;
+
+public class FakeReferenceSource {
```

Comment:
again, I think we should make this available to dependencies by moving it into our non-test source